1.9/installing/custom/configuration/configuration-parameters.md

```diff
@@ -56,12 +136,88 @@ This parameter specifies the name of your cluster.
+
+    *  `dcos_overlay_network` This group of parameters define an virtual network for DC/OS.  The default configuration of DC/OS provides an virtual network named `dcos` whose YAML configuration is as follows:
+
+        ```
+        dcos_overlay_network:
+            vtep_subnet: 44.128.0.0/20
+            vtep_mac_oui: 70:B3:D5:00:00:00
+            overlays:
+              - name: dcos
+                subnet: 9.0.0.0/8
+                prefix: 26
+        ```
+
+        *  `vtep_subnet` This parameter specifies a dedicated address space that is used for the VxLAN backend for the virtual network. This address space should not be routeable from outside the agents or master.
+        *  `vtep_mac_oui` This parameter specifies the MAC address of the interface connecting to it in the public node.
+            
+            **Important:** The last 3 bytes must be `00`.
+        *  __overlays__
+            *  `name` This parameter specifies the canonical name (see [limitations](/docs/1.9/networking/virtual-networks/) for constraints on naming virtual networks).
+            *  `subnet` This parameter specifies the subnet that is allocated to the virtual network.
+            *  `prefix` This parameter specifies the size of the subnet that is allocated to each agent and thus defines the number of agents on which the overlay can run. The size of the subnet is carved from the overlay subnet.
+
+ For more information see the [example](#overlay) and [documentation](/docs/1.9/networking/virtual-networks/).
+ 
+### <a name="dns-search"></a>dns_search
+This parameter specifies a space-separated list of domains that are tried when an unqualified domain is entered (e.g. domain searches that do not contain &#8216;.&#8217;). The Linux implementation of `/etc/resolv.conf` restricts the maximum number of domains to 6 and the maximum number of characters the setting can have to 256. For more information, see <a href="http://man7.org/linux/man-pages/man5/resolv.conf.5.html">man /etc/resolv.conf</a>.
+
+A `search` line with the specified contents is added to the `/etc/resolv.conf` file of every cluster host. `search` can do the same things as `domain` and is more extensible because multiple domains can be specified.
+
+In this example, `example.com` has public website `www.example.com` and all of the hosts in the datacenter have fully qualified domain names that end with `dc1.example.com`. One of the hosts in your datacenter has the hostname `foo.dc1.example.com`. If `dns_search` is set to &#8216;dc1.example.com example.com&#8217;, then every DC/OS host which does a name lookup of foo will get the A record for `foo.dc1.example.com`. If a machine looks up `www`, first `www.dc1.example.com` would be checked, but it does not exist, so the search would try the next domain, lookup `www.example.com`, find an A record, and then return it.
```

Comment:
@sascala seems like the jury is slightly in favor of using `lookup` as one word: 
- https://english.stackexchange.com/questions/290278/look-up-tables-look-up-tables-or-lookup-tables